enterprise_access/apps/api_client/ecommerce_client.py

```diff
@@ -0,0 +1,47 @@
+"""
+API client for calls to the ecommerce service.
+"""
+from django.conf import settings
+
+from enterprise_access.apps.api_client.base_oauth import BaseOAuthClient
+
+
+class EcommerceApiClient(BaseOAuthClient):
+    """
+    API client for calls to the ecommerce service.
+    """
+    api_base_url = settings.ECOMMERCE_URL + '/api/v2/'
+    enterprise_coupons_endpoint = api_base_url + 'enterprise/coupons/'
+
+    def get_coupon_overview(self, enterprise_uuid, coupon_id):
+        """
+        Call ecommerce API overview endpoint for data about a coupon.
```

Comment:
I assume this is copy pasta, that said, this is what we should shoot for docstring wise.